CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing to Libero
+
+We encourage contributions of any kind. By participating in this project, you agree to abide by the [code of conduct](https://libero.pub/code-of-conduct/).
+
+Contributions of code or documentation are covered by our [Contributor License Agreement (CLA)](https://libero.pub/governance/cla/). Please [sign the CLA](https://libero.pub/sign-cla) or make sure your employer has signed the CLA on your behalf. If you have questions about the CLA please email the team at libero@elifesciences.org.
+
+All Libero repositories will have further instructions in their own `CONTRIBUTING.md` file.
```

Comment:
or they won't have one, if this is used as the default?

Reply:
Ah yeah, that line 'should' be redundant once I merge this, hence best course of action is to remove it.